resolvedvelocities/synthetic_data/Field.py

```diff
@@ -0,0 +1,76 @@
+# Field.py
+import numpy as np
+from apexpy import Apex
+from scipy import interpolate
+import coord_convert as cc
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+
+import configparser
```

Comment:
Should be:

    try:
        import ConfigParser as configparser
    except ImportError:
        import configparser

for python2 compatibility.